src/main/java/uk/gov/hmcts/reform/sscs/util/DynamicListLanguageUtil.java

```diff
@@ -45,19 +44,6 @@ private List<DynamicListItem> generateInterpreterLanguage() {
 
     @NotNull
     public DynamicListItem getLanguageDynamicListItem(Language language) {

```

Comment:
Do we need a test I know it's in common but from util point we need to have a test ?

Reply:
There is already a test. I've cleaned it up a bit, so it should appear in the PR now